docs/checkit.js

```diff
@@ -162,36 +162,42 @@ CalendarAnalyzer.prototype.getCheckedDaysStreak = function () {
     // Returns the longest streak of checked days
     "use strict";
     
-    var currentStreak = 0,
+    var currentStreak = 1,
         checkedDaysArray = [],
-        lastDate,
         bestStreak = 0,
         i,
         currentDay,
         previousDay,
         dayDiff;
     
-    // place checkedDays object's keys into array and sort the array.
+    // Place checkedDays object's keys into array and sort the array.
     checkedDaysArray = Object.keys(this.calState.checkedDays);
     checkedDaysArray.sort();
-
-    lastDate = checkedDaysArray[0];
+    
+    previousDay = moment(checkedDaysArray[0], "YYYYMMDD");
```

Comment:
moment(undefined) is equivalent to moment(), which assumes the initial state to be the current date/time.